modules/fbs-core/api/src/main/scala/de/thm/ii/fbs/model/Course.scala

```diff
@@ -2,9 +2,10 @@ package de.thm.ii.fbs.model
 
 /**
   * A course
+  * @param semesterId semester in which the course is planed
   * @param name Name of the course
   * @param description The description of this course
   * @param visible The visibility of the course, false = invisible
   * @param id The id of the course, if 0, then none was assigned.
   */
-case class Course(name: String, description: String = "", visible: Boolean = true, id: Int = 0)
+case class Course(semesterId: Option[Int] = None, name: String, description: String = "", visible: Boolean = true, id: Int = 0)
```

Comment:
```suggestion
case class Course(name: String, description: String = "", visible: Boolean = true, id: Int = 0, semesterId: Option[Int] = None)
```
Arguments with default values should not be placed before arguments without default values.